bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -506,6 +525,24 @@ pub enum EncryptedMessage {
     Unknown,
 }
 
+impl EncryptedMessage {
+    fn new(msg: &matrix_sdk::room::timeline::EncryptedMessage) -> Self {
+        use matrix_sdk::room::timeline::EncryptedMessage as E;
```

Comment:
These single letter abbreviations make me think of generics when I see them, which makes me confused and double check what's going on. Not really needed but I would prefer something like `Encrypted` if `EncryptedMessage` is too long.

Reply:
`EncryptedMessage` is not too long, but I think it would be very confusing if `Self` is named `EncryptedMessage` and within its impl block there's a function where that name is used for something else.